docs/install.rst

```diff
@@ -169,13 +169,26 @@ Name                                 Notes
                                      audio and video streams, eg. YouTube 1080p+
 ==================================== ===========================================
 
+Using pycrypto and pycountry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Sorry about that @beardypig , forgot to adjust it after I made a change. Should be fixed now.